README.md

```diff
@@ -25,26 +24,39 @@ Players must locate batteries to keep their lights on, or face certain death in
 
 ### Multiplayer Networking
 
-DESCRIPTIONHERE
-    
+Our multiplayer networking implementation will synchronize the actions of up to 4 players across the network.
+One client player will also assume the role of the host. Clients will transmit actions to the server, which
+will resolve conflicts, simulate the world, and send state updates.
+
 ### Lighting Systems
 
-DESCRIPTIONHERE
+Dynamic 2D lighting will be implemented to provide a more interesting game atmosphere and add an element of
+resource/vision management to the mastery curve. Moving light sources, different light colors and color 
+blending, and different types of light sources (point, spotlight) will be implemented.
 
 ## Midterm Goals
 
-* GOAL1HERE
-* GOAL2HERE
-...
+* Player 4-direction movement
+* Player orients toward cursor
+* Player shooting
+* Enemies die when attacked
+* Enemy type that moves in straight line toward player
+* Players die when colliding with enemies
+* Phong lighting implementation
+* Point and cone lights illuminate enviornment withing radius
+* Server and client connection
 
 ## Final Goals
 
-* XX%: GOAL1HERE
-* XX%: GOAL2HERE
-* XX%: GOAL3HERE
-...
+* 15%: Players collide with surface
+* 05%: Players open/close doors
+* 30%: Complete level with fixed obstacles and enemy placements
```

Comment:
Too much for level design. Also need more details. How big of a level? How many obstacle types/instances?